### PR TITLE
Asset Metadata and sample data

### PR DESCRIPTION
**Pull request by hectorcorrea**
Updated Asset Metadata section with an example on how to define and set a date field. 

Added file `sandbox_data.md` as an sample on how to create the schema for the project and how to use it in an asset collection. We could use this as a guide to send the right request to MediaFlux through our Ruby code. We'll do that in a separate PR.

Screenshot 2023-08-31 at 4 42 30 PM

### CODE REVIEW

File: docs/sandbox_data.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    :element -name data_sponsor  -type string -index true -min-occurs 1 -max-occurs 1 -label "The person who takes primary responsibility for the project" \
+    :element -name data_manager  -type string -index true -min-occurs 1 -max-occurs 1 -label "The person who manages the day-to-day activities for the project" \
+    :element -name data_users_rw -type string -index true -min-occurs 0               -label "A person who has read and write access privileges to the project" \
+    :element -name data_users_ro -type string -index true -min-occurs 0               -label "A person who has read-only access privileges to the project" \
+    :element -name departments   -type string -index true -min-occurs 1               -label "The primary Princeton University department(s) affiliated with the project" \
+    :element -name created_on    -type date               -min-occurs 1 -max-occurs 1 -label "Timestamp project was created" \
+    :element -name created_by    -type string             -min-occurs 1 -max-occurs 1 -label "User that created the project" >
```
Comment (hectorcorrea) [inline docs/sandbox_data.md:39]: Uses the pre-MVP fields indicated in Matt's document (https://docs.google.com/document/d/1XjXkzhs5ngM5cmyrmTLaIxG6TF4jT7-HMsMrjj3akRw/edit)

File: docs/sandbox_data.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        :data_manager "kl37" \
+        :data_users_rw "hc8719" \
+        :data_users_rw "cac9" \
+        :data_users_ro "bs3097" \
+        :data_users_ro "jrg5" \
+        :departments "RDSS" \
+        :departments "HPC" \
+        :created_on "31-AUG-2023" \
+        :created_by "hc8719" > >
+
```
Comment (hectorcorrea) [inline docs/sandbox_data.md:73]: Sets the fields

File: docs/sandbox_data.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# Execute this script to populate a MediaFlux server with some sample data:
+# - a metadata schema
+# - a namespace
+# - a collection asset within the namespace
+# - set the collection asset with some values for the fields defined in the metadata schema
+#
+# You can run it from within Aterm with a command as follows:
```
Comment (carolyncole) [inline docs/sandbox_data.md:7]: Local aterm (not a web aterm)
```suggestion
# You can run it from within Local Aterm (not a Web Aterm) with a command as follows:
```

File: docs/sandbox_data.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+#
+# script.execute :in file://full/path/to/tiger-data-app/docs/sandbox_data.md
```
Comment (carolyncole) [inline docs/sandbox_data.md:9]:
```suggestion
# script.execute :in file://full/path/to/tiger-data-app/docs/sandbox_data.md
# 
# In the same aterm, to see the resultant collection with metadata run:
# asset.collection.list :namespace /tigerdata_ns02 :assets true
```

**Review by carolyncole — CHANGES_REQUESTED** [review]
@hectorcorrea just a couple small changes that would have made my life easier trying to run the script

**Comment by hectorcorrea** [comment]
@carolyncole I've accepted the changes and requested a re-review but it still shows as "changes requested" ¯\_(ツ)_/¯

**Pull request merged**